assets/css/_page.scss

```diff
@@ -0,0 +1,3 @@
+.c-page {
```

Comment:
This means adding a `c-page` class to all the pages' HTML. Another way to do this would be to make this a mixin and include it from the pages' CSS. Which do you think is a better place to do call this?